foundry/foundry.py

```diff
@@ -199,18 +199,23 @@ def load(self, name, download=True, globus=True, verbose=False, metadata=None, a
 
         return self
 
-    def list(self):
-        """List available Foundry data packages
+    def search(self, q=None, limit=None):
+        """Search available Foundry data packages
```

Comment:
iirc I believe we decided not to use the "package" language -- please change this to "datasets" or whatever you see fit